### PR TITLE
docs: readme.md to include creation of service account client

### PR DESCRIPTION
**Pull request by ArpitKotecha**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/storage/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by ArpitKotecha** [comment]
@googlebot I signed it!

**Comment by bcoe** [comment]
@ArpitKotecha :wave: we auto-generate this section of the README from `samples/quickstart.js` -- there's a file called `.readme-partials.yml`, that could be updated with an example of instantiating a client with service account JSON.

You can see it in action here: https://github.com/googleapis/release-please/blob/master/.readme-partials.yml#L18 (the `body` field would be where you add something).

**Comment by ArpitKotecha** [comment]
@bcoe 👋 Thanks for the info.

**Comment by frankyn** [comment]
@jkwlui could you help review this?

File: samples/quickstart.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ function main(
 
   // Creates a client
   const storage = new Storage();
+  // Creates a client from the Firebase Admin SDK service key
```
Comment (frankyn) [inline samples/quickstart.js]: Update to `Creates a client from a Google service account key.`

Comment (ArpitKotecha) [reply samples/quickstart.js]: Updated.

**Comment by ArpitKotecha** [comment]
@frankyn Help please. Why conventionalcommits.org check is not passing?

**Comment by frankyn** [comment]
Thanks @jkwlui, @ArpitKotecha you'll need to run `npm run fix`

**Comment by ArpitKotecha** [comment]
> npm run fix
Done.

**Pull request merged**